roles/installer/tasks/59_power_off_cluster_servers.yml

```diff
@@ -20,7 +20,7 @@
         - "{{ groups.workers | default([]) }}"
 
     - name: Power off hosts via IPMI
-      ansible.builtin.command: >
+      ansible.builtin.command: >-
```

Comment:
In other jobs I don't see the newline `\n` added to the command, last entries I see in the command are:
```
"power", "off"]
```

I'm curious if you see it and if you have a job example. Nonetheless I see no issues with removing it from the end.